modules/enterprise/binding/src/main/java/org/rhq/bindings/client/ResourceClientProxy.java

```diff
@@ -616,6 +618,13 @@ public void updateBackingContent(String filename, String displayVersion) {
 
             contentManager.deployPackagesWithNote(remoteClient.getSubject(), new int[] { resourceClientProxy.getId() },
                 new int[] { pv.getId() }, "CLI deployment request");
+            PackageDetailsKey keys = new PackageDetailsKey(
+                    oldPackage.getPackageVersion().getGeneralPackage().getName(),
+                    packageVersion,
+                    oldPackage.getPackageVersion().getGeneralPackage().getPackageType().getName(),
+                    oldPackage.getPackageVersion().getArchitecture().getName());
+            ResourcePackageDetails details = new ResourcePackageDetails(keys);
+            contentManager.mergePackage(remoteClient.getSubject(), resourceClientProxy.getId(), details);
```

Comment:
Is this really blocking? I mean, what happens if the agent takes ages to actually deploy the package? Are we now returning backing content that's not really actually deployed necessarily yet?